app/models/scsb_full_records_request.rb

```diff
@@ -0,0 +1,55 @@
+class ScsbFullRecordsRequest
+  attr_accessor :scsb_env, :email
+
+  def initialize(scsb_env, email)
+    @scsb_env = scsb_env
+    @email = email
+  end
+
+  def scsb_host
+    case scsb_env
+    when 'uat'
+      'https://uat-recap.htcinc.com:9093'
+    when 'production'
+      'https://scsb.recaplib.org:9093'
+    end
+  end
+
+  def scsb_conn
+    Faraday.new(url: scsb_host) do |faraday|
+      faraday.request(:url_encoded)
+    end
+  end
+
+  def scsb_request(institution_code)
+    response = scsb_conn.get do |req|
+      req.path = '/dataDump/exportDataDump'
+      req.params['collectionGroupIds'] = '1,2'
+      req.params['emailToAddress'] = email
+      req.params['fetchType'] = 10
+      req.params['imsDepositoryCodes'] = 'RECAP,HD'
```

Comment:
I wonder if this should be a parameter. Harvard is the only partner that has items in both facilities, for CUL and NYPL you'd only ask for "RECAP".